styleguide/source/_meta/_00-head.twig

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width" />
 
   <link rel="stylesheet" href="https://use.typekit.net/eie0akm.css" />
+  <link rel="stylesheet" href="../../assets/css/panels-ipe.css?{{ cacheBuster }}" media="all" />
```

Comment:
this should be converted to scss

Reply:
Disagree; this CSS file is just the CSS file that comes with the Panels IPE Drupal module. I only included it in the SG in order to be able to style on top of it. It's already present in D8.